src/main/scala/histogram/Histogram.scala

```diff
@@ -0,0 +1,48 @@
+package histogram
+
+import java.io.PrintWriter
+import scala.collection.parallel.CollectionConverters.ImmutableSeqIsParallelizable
+import scala.collection.parallel.ParSeq
+import scala.io.Source
+import scala.util.{Failure, Success, Try, Using}
+
+object Histogram {
+
+  def loadData(fileName: String): Try[ParSeq[String]] = {
+    Using(Source.fromFile(fileName)) {
+      bufferedSource => {
+        bufferedSource.getLines().toSeq.par.flatMap(_.split("[^\\d\\w]+"))
+      }
+    }
+  }
+
+  def saveData(fileName: String, counted: Seq[(String, Int)]): Unit = {
+    Using(new PrintWriter(fileName)) {
+      writer => {
+        counted.foreach(word => writer.write(s"${word._1} : ${word._2}\n"))
```

Comment:
It's better to use some meaningful naming instead of `word._1` and `word._2`, e.g.  :
`counted.foreach { case (word, qtty) => writer.write(s"$word : $qtty \n") }`

Reply:
Changed to proposed variant. Missed this form of tuple view. Will use in future tasks